internal/resources/configuration/describe.go

```diff
@@ -0,0 +1,108 @@
+// Copyright Mia srl
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configuration
+
+import "fmt"
+
+type DescribeConfiguration struct {
+	Config                     map[string]any `json:"config" yaml:"config"`
+	FastDataConfig             map[string]any `json:"fastDataConfig,omitempty" yaml:"fastDataConfig,omitempty"`
+	MicrofrontendPluginsConfig map[string]any `json:"microfrontendPluginsConfig,omitempty" yaml:"microfrontendPluginsConfig,omitempty"`
+	ExtensionsConfig           map[string]any `json:"extensionsConfig,omitempty" yaml:"extensionsConfig,omitempty"`
+}
```

Comment:
There is something peculiar about this struct to be a DescribeConfiguration instead of a standard Configuration?
If it can be generally used as a configuration we can reuse it in the import command